### PR TITLE
Update CHANGELOG for new external attacher release

### PR DESCRIPTION
**Pull request by davidz627**
/kind documentation

/assign @msau42 @jsafrane 

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.2.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# Changelog since v1.2.0
+
+## Action Required
+
+- processing of ControllerUnpublish errors has changed. CSI drivers SHOULD return success (0), when a deleted node or volume implies that the volume is detached from the node. The external attacher treats NotFound error as any other error and it assumes that the volume may still be attached to the node. Please check behavior of your CSI driver and fix it accordingly. ([#165](https://github.com/kubernetes-csi/external-attacher/pull/165), [@jsafrane](https://github.com/jsafrane))
+
+## Bug Fixes
+
+- Fix issue to actually translate backwards compatible access modes for CSI Migration ([#163](https://github.com/kubernetes-csi/external-attacher/pull/163), [@davidz627](https://github.com/davidz627))
```
Comment (msau42) [inline CHANGELOG-1.2.md]: was this fix backported to 1.2 branch?

File: CHANGELOG-1.2.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
+# Changelog since v1.2.0
+
+## Action Required
+
+- processing of ControllerUnpublish errors has changed. CSI drivers SHOULD return success (0), when a deleted node or volume implies that the volume is detached from the node. The external attacher treats NotFound error as any other error and it assumes that the volume may still be attached to the node. Please check behavior of your CSI driver and fix it accordingly. ([#165](https://github.com/kubernetes-csi/external-attacher/pull/165), [@jsafrane](https://github.com/jsafrane))
```
Comment (msau42) [inline CHANGELOG-1.2.md]: This release note needs to be modified to the cherry-pick PRs, which were made differently to be backwards compatible.  Namely that NotFound error is ignored, and all other errors are retried.  It still may require changes to the driver if it didn't implement the CSI spec correctly and return NotFound.  We should probably link to the cherry-pick PR instead as well.

**Comment by davidz627** [comment]
@msau42 fixed, I'm dumb and didn't specify the branch to the release-notes tool

**Pull request merged**